### PR TITLE
Add NL language

### PR DESCRIPTION
**Pull request by Cannonb4ll**
Should be all 🥳

### CODE REVIEW

File: resources/lang/nl/auth/login.php [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        'submit' => [
+            'label' => 'Inloggen',
+        ],
+
+    ],
+
+    'form' => [
+
+        'email' => [
+            'label' => 'Email adres',
```
[the remark this answers is not in the document]
Comment (Cannonb4ll) [reply resources/lang/nl/auth/login.php:16]: You can write it in many, many ways. There's huge discussions on how you should write it.

File: resources/lang/nl/auth/request-password.php [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+            'label' => 'Verstuur wachtwoord herstel link',
+        ],
+
+    ],
+
+    'form' => [
+
+        'email' => [
+            'hint' => 'Terug naar login',
+            'label' => 'Email adres',
```
[the remark this answers is not in the document]
Comment (Cannonb4ll) [reply resources/lang/nl/auth/request-password.php:17]: You can write it in many, many ways. There's huge discussions on how you should write it.

File: resources/lang/nl/auth/request-password.php [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    ],
+
+    'messages' => [
+
+        'throttled' => 'Teveel inlogpogingen. Probeer het opnieuw over :seconds seconden.',
+
+        'passwords' => [
+            'sent' => 'We hebben je wachtwoord reset link gemailed!',
+            'throttled' => 'Wacht voordat je het opnieuw probeert.',
+            'user' => 'We kunnen geen gebruiker vinden met dat email adres.',
```
[the remark this answers is not in the document]
Comment (Cannonb4ll) [reply resources/lang/nl/auth/request-password.php:29]: You can write it in many, many ways. There's huge discussions on how you should write it.

File: resources/lang/nl/auth/reset-password.php [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+        'passwordConfirmation' => [
+            'label' => 'Wachtwoord bevestiging',
+        ],
+
+    ],
+
+    'messages' => [
+
+        'passwords' => [
+            'throttled' => 'Wacht voordat je het opnieuw probeert..',
```
[the remark this answers is not in the document]
Comment (Cannonb4ll) [reply resources/lang/nl/auth/reset-password.php:32]: This is in the normal translation as well, its context is correct.

File: resources/lang/nl/resources/user-resource.php [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+return [
+
+    'form' => [
+
+        'avatar' => [
+            'label' => 'Avatar',
+        ],
+
+        'email' => [
+            'label' => 'Email adres',
```
[the remark this answers is not in the document]
Comment (Cannonb4ll) [reply resources/lang/nl/resources/user-resource.php:12]: You can write it in many, many ways. There's huge discussions on how you should write it.

File: resources/lang/nl/resources/user-resource.php [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+        ],
+
+    ],
+
+    'table' => [
+
+        'columns' => [
+
+            'email' => [
+                'label' => 'Email adres',
```
[the remark this answers is not in the document]
Comment (Cannonb4ll) [reply resources/lang/nl/resources/user-resource.php:65]: You can write it in many, many ways. There's huge discussions on how you should write it.

File: resources/lang/nl/auth/login.php (hunk 1 above)
[the remark this answers is not in the document]
Comment (dododedodonl) [reply resources/lang/nl/auth/login.php:16]: The reference I can find [De Taalunie](https://woordenlijst.org/#/?q=e-mailadres) says that with a dash and without space is the correct spelling, but everybody would know what is there

File: resources/lang/nl/auth/reset-password.php (hunk 4 above)
[the remark this answers is not in the document]
Comment (dododedodonl) [reply resources/lang/nl/auth/reset-password.php:32]: Did not double check that, double dots just looked weird (would expect single or three), my bad

[the remark this answers is not in the document]
Comment (danharrin) [reply resources/lang/nl/auth/reset-password.php:32]: Yeah I'm gonna check this across all translations 👍 Thanks for pointing it out.

**Pull request merged**